packages/javascript/bh-shared-ui/src/components/SearchCurrentNodes/types.ts

```diff
@@ -0,0 +1,12 @@
+export type FlatNode = GraphNode & { id: string };
+
+export type GraphNode = {
+    label: string;
+    kind: string;
+    objectId: string;
+    lastSeen: string;
+    isTierZero: boolean;
+    descendent_count?: number | null;
+};
+
+export type GraphNodes = Record<string, GraphNode>;
```

Comment:
We are using them once in `js-client-library`, in the `GraphData` type. My thought was that there is a change to pull the client library into the shared UI [being worked on here](https://github.com/SpecterOps/BloodHound/pull/47/files#diff-71c2c6df779e612c283a755cc2e47bb9a5f6c8f6b3a079344dd393dacd31cdd9R20), and this could be a temporary solution until we can just use those types directly. Otherwise I agree it is a bad idea to have type definitions duplicated across projects.

Reply:
Ah, I see. Yea, I'll leave this to your discretion. Mostly wanted to point out that we had it defined somewhere else already.